[2025-09-12T05:38:46.915+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-12T05:38:47.160+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_hourly_predictions.make_prediction scheduled__2025-09-11T16:00:00+00:00 [queued]>
[2025-09-12T05:38:47.202+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_hourly_predictions.make_prediction scheduled__2025-09-11T16:00:00+00:00 [queued]>
[2025-09-12T05:38:47.221+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-12T05:38:47.381+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): make_prediction> on 2025-09-11 16:00:00+00:00
[2025-09-12T05:38:47.685+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=1109) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-12T05:38:47.674+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_hourly_predictions', 'make_prediction', 'scheduled__2025-09-11T16:00:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/prediction_dag.py', '--cfg-path', '/tmp/tmpy_f9y5fb']
[2025-09-12T05:38:47.790+0000] {standard_task_runner.py:64} INFO - Started process 1114 to run task
[2025-09-12T05:38:47.800+0000] {standard_task_runner.py:91} INFO - Job 50: Subtask make_prediction
[2025-09-12T05:38:48.416+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_hourly_predictions.make_prediction scheduled__2025-09-11T16:00:00+00:00 [running]> on host c556a26b394a
[2025-09-12T05:38:49.193+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ops-team' AIRFLOW_CTX_DAG_ID='simple_hourly_predictions' AIRFLOW_CTX_TASK_ID='make_prediction' AIRFLOW_CTX_EXECUTION_DATE='2025-09-11T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-11T16:00:00+00:00'
[2025-09-12T05:38:49.226+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-12T05:38:49.347+0000] {logging_mixin.py:188} INFO - üîÆ Making delay prediction...
[2025-09-12T05:48:05.725+0000] {logging_mixin.py:188} INFO - ‚ùå Not enough recent data
[2025-09-12T05:48:07.194+0000] {python.py:237} INFO - Done. Returned value was: insufficient_data
[2025-09-12T05:48:07.195+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-12T05:48:07.252+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_hourly_predictions, task_id=make_prediction, run_id=scheduled__2025-09-11T16:00:00+00:00, execution_date=20250911T160000, start_date=20250912T053847, end_date=20250912T054807
[2025-09-12T05:48:07.405+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-12T05:48:07.594+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-12T05:48:07.597+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
